package.json

```diff
@@ -53,7 +53,7 @@
     "@types/react-test-renderer": "^16.9.1",
     "@types/sanitize-filename": "^1.6.3",
     "antlr4ts-cli": "^0.5.0-alpha.4",
-    "cypress": "^12.8.1",
+    "cypress": "^13.0.0",
```

Comment:
You can enforce cypress request 3.0 without upgrading - https://github.com/opensearch-project/dashboards-reporting/pull/174 I would recommend that unless you have tested all files already

Reply:
Also FTR will be 12 soon - I would recommend staying on same major version as them